<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_JackClients</class>
 <widget class="QDialog" name="Dialog_JackClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>468</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Jack Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30,30,30);
border-color: rgb(255, 236, 23);
color: rgb(255, 236, 170);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tree_widget_jack_input_clients">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>280</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="tree_widget_jack_output_clients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QCustomPlot" name="widget_jack_connection_lines" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>95</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30,30,30);
border-color: rgb(255, 236, 23);
color: rgb(255, 236, 170);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_jack_connections_connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>435</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_jack_connections_disconnect">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>435</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_jack_connections_dDisconnect_all">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>435</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Disconnect All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_jack_connections_refresh">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>435</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_jack_connections_expand_all">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>435</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Expand All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_box_auto_connect_all_jack_ports">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Auto Connect All Ports</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>widget_jack_connection_lines</zorder>
  <zorder>buttonBox</zorder>
  <zorder>tree_widget_jack_input_clients</zorder>
  <zorder>tree_widget_jack_output_clients</zorder>
  <zorder>push_button_jack_connections_connect</zorder>
  <zorder>push_button_jack_connections_disconnect</zorder>
  <zorder>push_button_jack_connections_dDisconnect_all</zorder>
  <zorder>push_button_jack_connections_refresh</zorder>
  <zorder>push_button_jack_connections_expand_all</zorder>
  <zorder>check_box_auto_connect_all_jack_ports</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_JackClients</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_JackClients</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
